### PR TITLE
Add 'welcome to Exercism' hints on 'Two Fer'

### PR DESCRIPTION
**Pull request by bitfield**
On the first exercise, give students some useful hints about how to progress quickly in the Go track, how to make sure their code is ready for mentoring, and so on.

### CODE REVIEW

File: gtpl/topic/hints.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    Like all automated tools, `golint` is not perfect, and there are occasions where you can safely ignore its advice, but you won't encounter any of them in Exercism exercises. Therefore, please follow all `golint`'s suggestions for now. Some of them may seem silly or excessive, but again, stick with it for now. You are building habits which will stand you in good stead later.
+
+    One of the first things `golint` is likely to complain about is missing documentation comments on your functions. Every Go function should have a comment above it explaining what the function does, what arguments it takes, and what it returns. Don't skip this or write silly comments to silence `golint`, like "Square yada yada yada." Sure, it's just an exercise, and you don't really need documentation, but you're learning the skills and habits to be a successful Go programmer, and good documentation is one of them.
```
Comment (tehsphinx) [inline gtpl/topic/hints.md]: I'd probably mention somewhere that this is only about exported functions. Otherwise students might freak ;)

Comment (bitfield) [reply gtpl/topic/hints.md]: Yes, I did consider that. But I think every Go function _should_ have a documentation comment. So I'm inclined to let that point stand.

**Comment by bitfield** [comment]
Any idea what causes this issue? I keep seeing it:

```
go: verifying github.com/kevinburke/go-bindata@v3.13.0+incompatible: checksum mismatch
	downloaded: h1:hThDhUBH4KjTyhfXfOgacEPfFBNjltnzl/xzfLfrPoQ=
	go.sum:     h1:+1zvIFm0AWO3wJmjRIcKV1cuwF1urt7WsJBhordma/k=
The command "go mod download" failed and exited with 1 during .
```

**Comment by tehsphinx** [comment]
I am not sure. Trying to find out why that is...

When I remove all cached `kevinburke` folders locally and remove the entries from `go.sum` and `go.mod` and then run:

```
go get -u github.com/kevinburke/go-bindata
go mod tidy
```

it recreates those entries the same they were before. But when pushing, travis complains about wrong checksums.

**Comment by tehsphinx** [comment]
I am updating go locally now due to this bug:
https://github.com/golang/go/issues/29278

**Comment by tehsphinx** [comment]
That did it...

Go versions pre 1.11.4 create a wrong checksum for the bindata package.

**Pull request merged**